src/conf.rs

```diff
@@ -0,0 +1,189 @@
+use std::{fmt, fs, io};
```

Comment:
perhaps this should be in utils?

Maybe we should restructure clippy in a different PR to move lint modules under `mod lints` or something.